apollo-router/src/executable.rs

```diff
@@ -159,7 +159,7 @@ pub struct Opt {
     )]
     log_level: String,
 
-    /// Reload configuration and schema files automatically.
+    /// Reload locally provided configuration and supergraph files automatically.  This only affects watching of local files and does not affect supergraphs and configuration provided by GraphOS through Uplink, which is always reloaded immediately.
```

Comment:
We kind of do right now with licenses, though it's not like "the whole configuraton" as it stands right now.  I think we can keep the door open for that in the future, because I can't imagine the behavior would be different without completely revisiting the UX.

Reply:
We're definitely considering if GraphOS would deliver (at least a base) configuration, rather than requiring it to be provided locally.

Reply:
That's fine. I didn't want to have a message that caused people to think configuration might be overriden or delivered via uplink unless we actually were going to do that.